playground/package.json

```diff
@@ -4,14 +4,15 @@
   "private": true,
   "dependencies": {
     "@stellar/elements": "^0.0.0",
+    "big.js": "^5.2.2",
```

Comment:
Is this right? Or should it be `bignumber.js`?

Reply:
Yeah, I ripped the tx previewer from StellarX which used big.js, it's probably not a big enough deal to port it.